actors/builtin/reward/expneg_test.go

```diff
@@ -0,0 +1,49 @@
+package reward
+
+import (
+	"bytes"
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/xorcare/golden"
```

Comment:
I like the approach, but this project has a super hard line minimal dependencies policy. Even though this is only used it a test right now, if we ever accidentally reference it it will become part of the set of things for which normal Go versioning semantics are not sufficient and we'll need to vendor, etc.

It look like we only need this for something fairly simple. Could I convince you to write a few more lines of code, inline the golden data here, and be able to drop the dependency?

Reply:
@anorth I plan using more golden tests for other computations.
Inlining is not the best as I use an external program to verify the precision of calculations (not possible in Go) and I import golden files as csv.

The `github.com/xorcare/golden` is itself is very small with the only dependency being `assertify` which we are already using.
Either way, I can also just write something like `golden.Assert`.

Reply:
I'll leave it to your judgement, knowing the situation. We can follow up to re-implement something like that ourselves later.